src/components/AvatarGroup/AvatarGroup.jsx

```diff
@@ -0,0 +1,144 @@
+import React, { useCallback } from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+import Avatar from "../Avatar/Avatar";
+import Counter from "../Counter/Counter";
+import Tooltip from "../Tooltip/Tooltip";
+import AvatarGroupCounterTooltipContainer from "./AvatarGroupCounterTooltipContainer";
+import styles from "./AvatarGroup.module.scss";
+
+const AvatarGroup = ({
+  className,
+  id,
+  children,
+  size,
+  type,
+  max,
+  counterProps,
+  counterTooltipCustomProps,
```

Comment:
why counterTooltipCustomProps and not counterTooltipProps, the 'custom' is redundant